frontend/src/components/UserTables/UnscheduledTable.tsx

```diff
@@ -34,88 +18,24 @@ const Table = ({ drivers }: TableProps) => {
   };
 
   const getUnscheduledRides = () => {
-    const today = moment(new Date()).format('YYYY-MM-DD');
-    fetch(`/rides?type=unscheduled&date=${today}`)
+    // uncomment to get the rides for only today
+    // const today = moment(new Date()).format('YYYY-MM-DD');
+    // fetch(`/rides?type=unscheduled&date=${today}`)
+    fetch(`/rides?type=unscheduled`)
```

Comment:
Should probably show "tomorrow's" unscheduled rides

Reply:
what's the reason for this?